javav2/example_code/elasticbeanstalk/src/main/java/com/aws/example/CreateApplication.java

```diff
@@ -0,0 +1,77 @@
+//snippet-sourcedescription:[CreateApplication.java demonstrates how to create an AWS Elastic Beanstalk application .]

```

Comment:
Remove extra space between "application" and the ending punctuation.

Reply:
fixed